types/selenium-webdriver/test/bidi/evaluateResult.ts

```diff
@@ -0,0 +1,28 @@
+import { error } from "console";
+import {
+    EvaluateResultException,
+    EvaluateResultSuccess,
+    ExceptionDetails,
+} from "selenium-webdriver/bidi/evaluateResult";
+
+function TestEvaluateResultSuccess() {
+    const evaluateResultSuccess = new EvaluateResultSuccess("asdf", "string");
+    if (evaluateResultSuccess.result !== "string") {
+        throw new error("EvaluateResultSuccess failure");
```

Comment:
```suggestion
        throw new Error("EvaluateResultSuccess failure");
```
`console.error` just logs, doesn't return anything